cmd/sessionstorage.go

```diff
@@ -0,0 +1,238 @@
+//
+// Copyright (c) 2023 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package cmd
+
+import (
+	authApi "github.com/SSHcom/privx-sdk-go/api/auth"
+	"github.com/spf13/cobra"
+)
+
+type sessionStorageOptions struct {
+	userID    string
+	sourceID  string
+	sessionID string
+	sortkey   string
+	sortdir   string
+	offset    int
+	limit     int
+}
+
+func init() {
+	rootCmd.AddCommand(sessionStorageCmd())
+}
+
+//
+//
+func sessionStorageCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sessions",
+		Short: "List and manage sessions",
+		Long:  `List and manage sessions`,
+		Example: `
+	privx-cli sessions [access flags]
+		`,
+		SilenceUsage: true,
+	}
+
+	cmd.AddCommand(userSessionsShowCmd())
+	cmd.AddCommand(sourceSessionsShowCmd())
+	cmd.AddCommand(sessionsSearchCmd())
+	cmd.AddCommand(terminateSessionCmd())
+	cmd.AddCommand(terminateUserSessionsCmd())
+
+	return cmd
+}
+
+//
+//
+func userSessionsShowCmd() *cobra.Command {
+	options := sessionStorageOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "show-by-user",
```

Comment:
We could maybe simplify the CLI by having only a"show" sub-command and get rid of "show-by-user" and "show-by-source". 
Instead we would use flags like for example ''--user-id" and "--source-id" to determine what API endpoint we should call instead depending what flag is used.

Reply:
refactored